src/charts/ViewManager.ts

```
@@ -212,19 +215,14 @@ class ViewManager {
-                if (Object.keys(viewData.dataSources)?.length === 1) {
-                    const name = Object.keys(viewData.dataSources)?.[0];
-                    state.view.initialCharts[name] = [];
-                    state.view.dataSources[name] = {};
-                } else {
-                    for (const ds in dsIndex) {
-                        if (checkedDs[ds]) {
-                            state.view.initialCharts[ds] = [];
-                            state.view.dataSources[ds] = {};
-                        }
+                //add the required datasources (checkedIds) to the view
+                for (let name in checkedDs){
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'name' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)